File: sources/frontend/src/models/Project.ts
import { User } from "src/models/User";
import { Resources } from "src/models/Resources";

export interface Project {
  id: string;
  name: string;
  active: boolean;
  admins: User[];
  payUser: User;
  members: User[];
  quota: Resources;

  userQuotas: {[id: string]: Resources};

}

File: sources/backend/API/Models/Identity/Projects/Project.cs
using AcademyCloud.API.Models.Common;
using AcademyCloud.API.Models.Identity.Common;
using System.Collections.Generic;

namespace AcademyCloud.API.Models.Identity.Projects
{
    public class Project
    {
        public string Id { get; set; }
        public string Name { get; set; }

        public bool Active { get; set; }

        public IEnumerable<User> Admins { get; set; }
        public IEnumerable<User> Members { get; set; }

        public User PayUser { get; set; }

        public Resources Quota { get; set; }

        public IDictionary<string, Resources> UserQuotas { get; set; }
    }
}